docs/_research/findings/1-flow-diagrams.md

```diff
@@ -1,50 +0,0 @@
----
-title: 1 - Flow diagrams
-lead: >
-  Recommendations from usability testing some user flow diagrams
----
-
-## Summary
-
-In an effort to better communicate how the login.gov service communicates with third party providers and managers user data, the team created two different graphics to demonstrate how the login services work. We tested these two diagrams with a few users to see if they would be understandable to various stakeholders.
-
-![data flow model](/post-images/login--data-flow-model.svg)
```

Comment:
Can we remove this file as well, since it's now unused?

Reply:
Done 8899f2f2792d8dda701566a53707b1c674a6611d